app/scripts/modules/core/src/managed/ArtifactsList.tsx

```diff
@@ -22,7 +23,7 @@ export function ArtifactsList({ artifacts, artifactSelected }: IArtifactsListPro
             clickHandler={artifactSelected}
             version={version}
             name={name}
-            sha="abc123"
+            sha=""
```

Comment:
maybe just give this the axe for now? 🔪

Reply:
Oh ya, since I'm now passing in the full version object

Reply:
> passing in the full version object
Whoops left that part and some other things unpushed

Reply:
still looks great to me